frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -766,8 +768,8 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
                 }
                 const snapshot = snapshots[currIndex]
                 return {
-                    width: snapshot.data?.['width'],
-                    height: snapshot.data?.['height'],
+                    width: (snapshot.data as any).width,
+                    height: (snapshot.data as any).height,
```

Comment:
is this less safe now we don't have `?.` so if we try and access undefined data we'll have an error?

why did this change? might be pointing us at something else we need to fix